lib/vfscore/automount.c

```diff
@@ -50,18 +50,20 @@
 #include <uk/plat/memory.h>
 #include <sys/stat.h>
 #include <vfscore/mount.h>
-
-#ifdef CONFIG_LIBVFSCORE_AUTOMOUNT_ROOTFS
 #include <errno.h>
 #include <uk/config.h>
 #include <uk/arch/types.h>
-#endif /* CONFIG_LIBVFSCORE_AUTOMOUNT_ROOTFS */
 
-#if CONFIG_LIBVFSCORE_FSTAB
+#if CONFIG_LIBVFSCORE_AUTOMOUNT_UP
 #define LIBVFSCORE_FSTAB_VOLUME_ARGS_SEP			':'
 #define LIBVFSCORE_FSTAB_UKOPTS_ARGS_SEP			','
```

Comment:
If we do this, compiling with `Compiled-in filesystem table` and without `User-provided filesystem table` config options will fail with `vfscore/automount.c:466:41: error: LIBVFSCORE_FSTAB_UKOPTS_ARGS_SEP’ undeclared`.
Is there any reason to add these in a guard and not just always define them?

Reply:
Ouh, good point, there is one tiny mistake then. But you have actually good point, they don't harm if they are unused, no memory will be used. I'll remove this guard. Thanks for spotting this.